pkg/apiserver/index.go

```diff
@@ -26,10 +26,6 @@ import (
 // handleIndex is the root index page for Kubernetes.
 func handleIndex(req *restful.Request, resp *restful.Response) {
 	// TODO: use restful's Request/Response methods
-	if req.Request.URL.Path != "/" && req.Request.URL.Path != "/index.html" {
```

Comment:
Please set up a [redirect](http://golang.org/pkg/net/http/#Redirect) from `/` to something reasonable: maybe healthz?